### ISSUE TITLE
Focus getting lost on tool bar item action in Ionic framework app

### ISSUE THREAD

**Issue by induri** [fraywing/textAngular#1195]
Hi,

I've integrated textAngular in an Ionic App.
Everything working fine in Android, but in IOS the focus in text editor getting lost when I click any item in toolbar.
  <div id="demo-editor" text-angular ng-model="formData.content" name="demo-editor"
        ta-text-editor-class="form-control myform2-height" 
        ta-html-editor-class="form-control myform2-height"
        ta-target-toolbars='toolbar1'                    
    >
    </div>
    <text-angular-toolbar name="toolbar1" class="toolbar" ta-toolbar="[['h1','bold','italics','underline', 'quote'],['ul','ol','insertLink','insertImage', 'insertVideo']]"></text-angular-toolbar>

I tried to restore selection using rangy in action function like
action: function(deferred, restoreSelection){
...wrap function call
restoreSelection();
}

Is it issue with TextAngular integration with Ionic View  or issue with in Ionic framework.
Please help.

**Comment by JoelParke** [comment]
Have you made any progress on this issue?

**Comment by induri** [comment]
Yes, we've to reset the focus from parent, i.e. on focus event in ionic view, i'm resetting focus to iframe view which hosts text angular editor.

-Prakash.

**Comment by azachar** [comment]
@induri Could you please provide with PR for this issue? Or at least a code that can be copy paste and patched? Thankx
